prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/TranslationFunctions.kt

```diff
@@ -0,0 +1,68 @@
+package gov.cdc.prime.router.fhirengine.translation.hl7.utils
+
+import ca.uhn.fhir.model.api.TemporalPrecisionEnum
+import ca.uhn.hl7v2.model.v251.datatype.DTM
+import org.hl7.fhir.r4.model.BaseDateTimeType
+
+/**
+ * This interface contains the required method signatures required to implement FHIR transloation
+ * functions. It contains the default dateTime convertion function.
+ */
+interface TranslationFunctions {
```

Comment:
Why do we need two `convertDateTime` functions? I think we should only have one, to simplify the usage, especially since they both take in the same parameters. After parsing the parameters, this function should basically just call methods from DateUtilities.

The fact that the overriding function is called "custom..." makes me think we are not using this pattern correctly.

Reply:
Hmmm, you get the point here.  I will discuss this with @victor-chaparro and see whether we only need one or both.

Reply:
I discussed with Victor, and your changes make sense to me now. This is so we can eventually move the interface, which includes default behavior that anyone can use, to its own library. The extending function contains logic specific to ReportStream that wouldn't make sense in the library.